src/LinuxTracingIntegrationTests/LinuxTracingIntegrationTest.cpp

```diff
@@ -758,14 +758,6 @@ void VerifyCallstackSamplesWithOuterAndInnerFunction(
       // address and the caller address should match the "outer" function's address.
       if (callstack.pcs(pc_index) >= inner_function_virtual_address_range.first &&
           callstack.pcs(pc_index) <= inner_function_virtual_address_range.second) {
-        // Frame-pointer unwinding skips the second innermost frame if the sample fell on `push rbp`
```

Comment:
Have you run the affected tests at least, say, 1000 times? Just to be sure.

Reply:
There still seem to be an issue. I look into this right now.

Reply:
there are actually also quite large frames. I changed the size to 512 bytes. Still, there are some samples that gets discarded then, so I adjusted the test to also except this error kind.

Reply:
Exactly. I've seen a sample where rbp and rsp was more than 1000 bytes apart. Also I've seen a sample, where the difference was negative (probably by rbp not being set for whatever reason).

Reply:
Do you think that the cases with such big frames (for example 336) were actually caused by a big frame, or could it be that rbp was just completely wrong? In other words, is it actually worth it to increase from 128 to 512? I'm wondering because increasing to 512 B has a performance impact (for example with regard to the performance impact investigation you had done on this).

Reply:
As discussed offline. Keep it as it for now. We need to do some further experiments anyways.